legal/src/main/resources/legal/DEPENDENCIES

```diff
@@ -144,10 +172,13 @@ maven/mavencentral/org.jboss.logmanager/jboss-logmanager-embedded/1.0.9, Apache-
 maven/mavencentral/org.jboss.logmanager/log4j2-jboss-logmanager/1.0.0.Beta1, Apache-2.0, approved, clearlydefined
 maven/mavencentral/org.jboss.slf4j/slf4j-jboss-logmanager/1.1.0.Final, Apache-2.0, approved, clearlydefined
 maven/mavencentral/org.jboss.spec.javax.transaction/jboss-transaction-api_1.2_spec/1.1.1.Final, GPL-2.0-only WITH Classpath-exception-2.0 OR CDDL-1.0, approved, clearlydefined
+maven/mavencentral/org.jboss.spec.javax.xml.bind/jboss-jaxb-api_2.3_spec/2.0.0.Final, NOASSERTION, restricted, clearlydefined
```

Comment:
A restricted dependency entry got added here. I'm looking into this.

Reply:
I'll create a ClearlyDefined PR for this and update the DEPENDENCIES file later.

Reply:
With the dependency changes in #2847, this entry isn't there anymore, so it looks like there is no further action needed here with regards to the upcoming Hono 1.10.0.